Tutorials/Heisenberg1d/heisenberg1d.py

```diff
@@ -35,7 +35,7 @@
 #defining the wave function
 pars['Machine']={
     'Name'           : 'RbmSpinSymm',
-    'Alpha'          : 1.0,
+    'Alpha'          : 1,
```

Comment:
Is there a reason for this change?

Reply:
Alpha is supposed to be an integer in that case, I guess the change makes sense

Reply:
Ah, okay.